### ISSUE TITLE
📚 Documentation: Tutorial for Appwrite with Typescript

### ISSUE THREAD

**Issue by gewenyu99** [appwrite/website#95]
### Goal
Tutorial for beginners to learn to set up Appwrite Web SDK with Typescript.

### Content
1. Step by step from setup. 
1. Format should be similar to ` /docs/tutorials/sveltekit`
1. Should contain the following content
      1. Why typescript
      1. Project setup
      1. Request and response models
      1. One or two example request/responses

### How to contribute
Pitch an outline detailing each page you plan to add and headings within, and ask to be assigned. Open a PR **after** the outline has been approved.

The Appwrite team will collaborate with you on the outline and PR.

Thanks for taking the time to make our documentation better 🙏

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by PrerakMathur20** [comment]
Hey, I would like to work on this please.

**Comment by Mann-tech13** [comment]
I would like to work on this issue.

**Comment by Haimantika** [comment]
> Hey, I would like to work on this please.

Hi @PrerakMathur20 have assigned the issue to you! Thank you for showing interest in contributing to Appwrite! Happy Hacktoberfest 🎃

Notes:

Please update us with your progress every 3 days, so that we know that you are working on it.
Join us on Discord - https://appwrite.io/discord to chat about Hacktoberfest and Appwrite!

**Comment by Haimantika** [comment]
> I would like to work on this issue.

Hi, we assign issues on first-come first -serve basis. Have assigned the issue to Prerak, if he is unresponsive, we will assign it to you.

**Comment by gewenyu99** [comment]
@PrerakMathur20 Please provide a detailed outline on:

1. Each page and title
2. Each pages headings
3. A short description for what each heading describes and what it explains.

**Comment by gewenyu99** [comment]
Thanks!

**Comment by suryanshsoni120** [comment]
If the issue hasn't been resolved, I would like to work on this. Can you assign it to me?

**Comment by PrerakMathur20** [comment]
> @PrerakMathur20 Please provide a detailed outline on:
> 
> 1. Each page and title
> 2. Each pages headings
> 3. A short description for what each heading describes and what it explains.

Sure. Thanks for the update. I'm working on it. I'll create a PR asap.

**Comment by Gilbishkosma** [comment]
hi @gewenyu99  @Haimantika , i want to contribute to this one, you can assign it to me if no one else is working on it, thanks.

**Comment by gewenyu99** [comment]
> > @PrerakMathur20 Please provide a detailed outline on:
> > 
> > 1. Each page and title
> > 2. Each pages headings
> > 3. A short description for what each heading describes and what it explains.
> 
> Sure. Thanks for the update. I'm working on it. I'll create a PR asap.

Please create an outline before submitting a PR, thanks!

**Comment by JayanthT7** [comment]
Hi @Haimantika please assign one good first issue labelled hactoberfest which I can contribute on first time basis . I am starting my journey in open source please do help

**Comment by PrerakMathur20** [comment]
I have a rough outline for the Tutorial prepared. Should I add it in the comments here or on the Discord?

**Comment by Haimantika** [comment]
> > > > @PrerakMathur20 Please provide a detailed outline on:
> > > > 
> > > > 1. Each page and title
> > > > 2. Each pages headings
> > > > 3. A short description for what each heading describes and what it explains.
> > > 
> > > 
> > > Sure. Thanks for the update. I'm working on it. I'll create a PR asap.
> > 
> > 
> > Please create an outline before submitting a PR, thanks!
> 
> I have a rough outline for the Tutorial prepared. Should I add it in the comments here or on the Discord?

Add it here :)

**Comment by PrerakMathur20** [comment]
Please take a look [Build an ideas Tracker with TypeScript](https://docs.google.com/document/d/14Rn13zV8-o1KXFqudkRO79Zv9BlSQRxASZ3GC_oMcAI/edit?usp=sharing).

**Comment by gewenyu99** [comment]
> Please take a look [Build an ideas Tracker with TypeScript](https://docs.google.com/document/d/14Rn13zV8-o1KXFqudkRO79Zv9BlSQRxASZ3GC_oMcAI/edit?usp=sharing).

Can you open a PR instead? This is really hard to follow. Thanks!

**Comment by PrerakMathur20** [comment]
Sure, I'm working on it already. I'll open it as soon as I'm finished.

**Comment by akash47angadi** [comment]
@Haimantika I would like to work on this. Can you assign it to me?

**Comment by Haimantika** [comment]
> Sure, I'm working on it already. I'll open it as soon as I'm finished.

Are you still working on it? We will be reassigning the issue to the next person soon due to inactivity from your end.

**Comment by PrerakMathur20** [comment]
Hey @Haimantika , I'm almost done with the task. Will try to create the PR by EOD.

**Comment by tessamero** [comment]
@PrerakMathur20 the PR is awaiting your follow up so we can merge it.

We will close the PR if inactive soon.

**Comment by tessamero** [comment]
@PrerakMathur20 closing your PR on Friday if there is no follow up

**Comment by PrerakMathur20** [comment]
Apologies for the delay.

I have made a few changes in the pr which need to be reviewed.

**Comment by eldadfux** [comment]
Starting today this repo is archived. Please raise any issues with the new Appwrite website on the main platform repo: https://github.com/appwrite/appwrite
